provider/kubernetes/annotations.go

```diff
@@ -9,6 +9,10 @@ const (
 	annotationKubernetesAuthRealm                 = "ingress.kubernetes.io/auth-realm"
 	annotationKubernetesAuthType                  = "ingress.kubernetes.io/auth-type"
 	annotationKubernetesAuthSecret                = "ingress.kubernetes.io/auth-secret"
+	annotationKubernetesForwardAuthURL            = "ingress.kubernetes.io/auth-forward-url"
+	annotationKubernetesForwardAuthTrustHeaders   = "ingress.kubernetes.io/auth-forward-trust-headers"
+	annotationKubernetesForwardAuthTLSCert        = "ingress.kubernetes.io/auth-forward-tls-cert"
+	annotationKubernetesForwardAuthTLSKey         = "ingress.kubernetes.io/auth-forward-tls-key"
```

Comment:
Private keys should never be used in annotations.

These should be in secrets, and referenced via a `auth-tls-secret` annotation, just like with basic auth :)